config/locales/models/activity.en.yml

```diff
@@ -77,7 +77,7 @@ en:
         stage: Stage
         status: Status
         third_party_projects: Third-party projects
-        title: Title
+        title: Programme name
```

Comment:
Again this should be `%{level} name` because it might not always be a programme.

Reply:
Should I do these again in another pull request?

Reply:
If you have time, that would be super helpful.